### PR TITLE
fix(all): multiple findings and fixes

### PR DESCRIPTION
**Pull request by Wondertan**
Multiple independent commits that just fix minor things. Happy to answer any questions about each, if any.

### CODE REVIEW

File: service/share/light_availability.go [hunk 1]
```diff
@@ -18,13 +18,13 @@ var DefaultSampleAmount = 16
 // its availability. It is assumed that there are a lot of lightAvailability instances
 // on the network doing sampling over the same Root to collectively verify its availability.
 type lightAvailability struct {
-	getter blockservice.BlockGetter
+	bserv blockservice.BlockService
 }
 
 // NewLightAvailability creates a new light Availability.
-func NewLightAvailability(getter blockservice.BlockGetter) Availability {
+func NewLightAvailability(getter blockservice.BlockService) Availability {
```
Comment (renaynay) [inline service/share/light_availability.go:25]: Nit: should then rename also the param? --> `bserv`

**Comment by Wondertan** [comment]
[711e86a](https://github.com/celestiaorg/celestia-node/pull/807/commits/711e86ae9d94bacd0777c82d303bcf3ffb0f61f3), [ba8637f](https://github.com/celestiaorg/celestia-node/pull/807/commits/ba8637f345711cb4a6b509c7f8e38fa2c3af50a2) and [042ebe1](https://github.com/celestiaorg/celestia-node/pull/807/commits/042ebe1ae2c9f7337ef40032fbf3ef821495fb51) are basically fix of artifacts from #730

File: service/share/light_availability.go (hunk 1 above)
Comment (Wondertan) [inline service/share/light_availability.go:25]:
```suggestion
func NewLightAvailability(bserv blockservice.BlockService) Availability {
```

File: service/share/light_availability.go [hunk 2]
```diff
@@ -18,13 +18,13 @@ var DefaultSampleAmount = 16
...
 	return &lightAvailability{
-		getter: getter,
+		bserv: getter,
```
Comment (Wondertan) [inline service/share/light_availability.go:27]:
```suggestion
		bserv: bserv,
```

**Review by Wondertan — COMMENTED** [review]
self suggestions to quickly apply the change

**Comment by renaynay** [comment]
merge conflict @Wondertan

**Comment by Wondertan** [comment]
Fixed

**Review by vgonkivs — APPROVED** [review]

**Review by liamsi — APPROVED** [review]
👏🏼

**Pull request merged**